README.md

```diff
@@ -60,6 +60,10 @@ Publications
 
 Please read the reference documentation
 
+   Bauer, R., Grooß, J.-U., Ungermann, J., Bär, M., Geldenhuys, M., and Hoffmann, L.: The Mission Support
+   System (MSS v7.0.4) and its use in planning for the SouthTRAC aircraft campaign, Geosci.
+   Model Dev., 15, 8983–8997, https://doi.org/10.5194/gmd-15-8983-2022, 2022.
+
    Rautenhaus, M., Bauer, G., and Doernbrack, A.: A web service based
    tool to plan atmospheric research flights, Geosci. Model Dev., 5,
    55-71, doi:10.5194/gmd-5-55-2012, 2012.
```

Comment:
please update this doi address  too. 

from 
doi:10.5194/gmd-5-55-2012

to
https://doi.org/10.5194/gmd-5-55-2012

also in publications.rst

Reply:
the NOTICE file also has the old URL regarding the 2012 release, shall I update that too?